src/ArcxAnalyticsSdk.ts

```diff
@@ -253,6 +258,19 @@ export class ArcxAnalyticsSdk {
     return true
   }
 
+  private _trackClicks() {
+    window.addEventListener('click', (event) => {
+      this._onClick(event)
+    })
+  }
+
+  private _onClick(event: any) {
```

Comment:
Can we add a type here?

Reply:
No, these type miss needed attributes, which exist in browsers which i checked

Reply:
Looks like we _can_ add types. I made a quick pr to merge into this one, lmk what you think: https://github.com/arcxmoney/analytics-sdk/pull/74